xml/System.Reflection/TypeDelegator.xml

```diff
@@ -1048,9 +1048,23 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>When overridden in a derived class, returns the calling conventions of the current function pointer <see cref="T:System.Type" />.</summary>
+        <returns>
+          <para>An array of <see cref="T:System.Type" /> objects representing all the calling conventions for the current function pointer <see cref="T:System.Type" />.</para>
+          <para>-or-</para>
+          <para>An empty array of type <see cref="T:System.Type" />, if no calling conventions are defined for the current function pointer <see cref="T:System.Type" />.</para>
+          <para>-or-</para>
+          <para>An empty array of type <see cref="T:System.Type" />, if the current function pointer <see cref="T:System.Type" /> is not a modified <see cref="T:System.Type" />. A modified <see cref="T:System.Type" /> is obtained from <see cref="M:System.Reflection.FieldInfo.GetModifiedFieldType" />, <see cref="M:System.Reflection.PropertyInfo.GetModifiedPropertyType" />, or <see cref="M:System.Reflection.ParameterInfo.GetModifiedParameterType" />.</para>
+        </returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ This method is provided for designers of managed compilers. For more information on custom modifiers, see classes in the <see cref="N:System.Runtime.CompilerServices"/> namespace.
+
+ ]]></format>
+        </remarks>
```

Comment:
To be able to use `see cref`, these remarks need to be moved from markdown to xml:

```suggestion
        <remarks>This method is provided for designers of managed compilers. For more information on custom modifiers, see classes in the <see cref="N:System.Runtime.CompilerServices"/> namespace.</remarks>
```